frontend/public/components/marketplace/kubernetes-marketplace.jsx

```diff
@@ -60,24 +72,32 @@ class MarketplaceListPage extends React.Component {
     }
   }
 
+  toggleOpen(item) {
+    this.setState(prevState => {
+      const selectedTile = prevState.selectedTile === item ? null : item;
+      return { selectedTile };
+    });
```

Comment:
Since the overlay is modal, I don't believe the user could ever click on a tile to toggle the overlay closed. This really just needs to be `openOverlay` and set the state.selectedTile (selectedItem?).

Reply:
Yeah, I think it's more readable/reliable to have separate functions.